modules/apps/commerce/commerce-report-impl/bnd.bnd

```diff
@@ -36,12 +32,6 @@ Import-Package:\
 	\
 	!javax.rmi.*,\
 	\
-	!javax.swing.*,\
-	\
-	!jline.*,\
-	\
-	!junit.*,\
-	\
 	!mondrian.*,\
```

Comment:
@marianoalvarosaiz, after the fix, the package **javax.swing** can be imported to Import-Package in the module `com.liferay.commerce.report.impl`. This is not consistent with the original. It may indirect import by other libs. Please check the kind of the case. For the kind of case, you may add a new commit to explain this. We need to keep it existed in the module `com.liferay.commerce.report.impl`.

Reply:
I've explained now in the commit message why it is needed (due to other included libraries).